docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -97,18 +115,44 @@ cluster. This can help if you encounter a bug which renders the user interface p
 It might also help if you have issues with connecting to an active cluster that don't happen in the
 Web UI.
 
+<Tabs>
+<TabItem label="macOS">
 To force the app to log you out of all clusters, close the app and remove the `~/Library/Application
-Support/Teleport Connect/tsh` folder. Removing the file under `~/Library/Application
+Support/Teleport Connect/tsh` folder. Removing the file `~/Library/Application
 Support/Teleport Connect/app_state.json` will clear all remembered tabs and connections.
+</TabItem>
+<TabItem label="Linux">
+To force the app to log you out of all clusters, close the app and remove the `~/.config/Teleport
+Connect/tsh` folder. Removing the file `/.config/Teleport Connect/app_state.json` will clear
+all remembered tabs and connections.
+</TabItem>
+<TabItem label="Windows">
+To force the app to log you out of all clusters, close the app and remove the
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\tsh` folder. Removing the file
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\app_state.json` will clear all remembered tabs
```

Comment:
Same consideration to use `%AppData%` as in my other comment.

Reply:
I think utilizing `%UserName%` here makes more sense, to give readers a better understanding of file paths. `%AppData%` makes sense when we're just trying to run a command and path knowledge doesn't matter as much.
```suggestion
`C:\Users\%UserName%\AppData\Roaming\Teleport Connect\tsh` folder. Removing the file
`C:\Users\%UserName%\AppData\Roaming\Teleport Connect\app_state.json` will clear all remembered tabs
```